WebHookHandlers/Telegram/Actions/IAction.cs

```diff
@@ -0,0 +1,6 @@
+namespace JewishBot.WebHookHandlers.Telegram.Actions
+{
+    internal interface IAction
```

Comment:
From what I understood after having a look at classes that implement this interface, it might make sense to put a signature of HandleAsync here, like so:

```
void HandleAsync(long chatId, string[] args)
```

Or remove it at all. Maybe I'm wrong.